src/vcpkg/help.cpp

```diff
@@ -49,58 +52,35 @@ namespace vcpkg::Help
     static void help_topic_versioning(const VcpkgPaths&)
     {
         HelpTableFormatter tbl;
-        tbl.text("Versioning allows you to deterministically control the precise revisions of dependencies used by "
-                 "your project from within your manifest file.");
+        tbl.text(msg::format(msgVersioning));
         tbl.blank();
         tbl.blank();
-        tbl.header("Versions in vcpkg come in four primary flavors");
-        tbl.format("version", "A dot-separated sequence of numbers (1.2.3.4)");
-        tbl.format("version-date", "A date (2021-01-01.5)");
-        tbl.format("version-semver", "A Semantic Version 2.0 (2.1.0-rc2)");
-        tbl.format("version-string", "An exact, incomparable version (Vista)");
+        tbl.header(msg::format(msgVersionFourFlavors));
+        tbl.format("version", msg::format(msgVersionHelp));
+        tbl.format("version-date", msg::format(msgVersionDateHelp));
+        tbl.format("version-semver", msg::format(msgVersionSemverHelp));
+        tbl.format("version-string", msg::format(msgVersionStringHelp));
         tbl.blank();
-        tbl.text("Each version additionally has a \"port-version\" which is a nonnegative integer. When rendered as "
-                 "text, the port version (if nonzero) is added as a suffix to the primary version text separated by a "
-                 "hash (#). Port-versions are sorted lexographically after the primary version text, for example:");
+        tbl.text(msg::format(msgPortVersionHelp));
         tbl.blank();
         tbl.blank();
         tbl.text("    1.0.0 < 1.0.0#1 < 1.0.1 < 1.0.1#5 < 2.0.0");
```

Comment:
I think this part should be part of msgPortVersionHelp, even if it means the blank lines are removed.